docs/source/advancedcommands.rst

```diff
@@ -233,3 +233,26 @@ Example Usage
 The ``normsub`` argument specifies an expected value for your solution to normalize the different :math:`g_i` (you can also do this by hand). The feasibility of the problem should not depend on the normalization, but the spacing of the sweep will.
 
 The ``sweep`` argument specifies what points between 0 and 1 you wish to sample the weights at. If you want different resolutions or spacings for different weights, the ``sweeps`` argument accepts a list of sweep arrays.
+
+
+Debugging
+=========
+
+Unbounded variables
+-------------------
+In some cases a solver will return ``unknown`` when a free variable is
+unbounded by the model. ``gpkit.constraints.bounded.BoundedConstraintSet`` is a
+simple tool that attempts to detect unbounded variables by adding extremely
+large upper bounds and extremely small lower bounds to all variables in a ConstraintSet.
+
+When a model if with a BoundedConstraintSet is solved, it checks whether any variables slid off to the bounds, notes this in the solution dictionary and prints a warning (if verbosity is greater than 0).
+
+For example, Mosek returns unknown when attempting to solve the following model:
```

Comment:
I think the above should mention dual infeasibility -- in particular,

 - A model that is DUAL_INFEAS_CER is always a good candidate for debugging with a BoundedConstraintSet
 - A model that is UNKNOWN is sometimes a possible to debug with a BoundedConstraintSet -- when the unknown is actually caused by dual infeasibility undetected by the solver.